2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -32,6 +32,10 @@ object Dependencies {
     val slf4j                = "1.7.26"
     val config               = "1.3.4"
     val prometheus           = "0.6.0"
+    /** TODO Make sure cbor version stays aligned with jackson-core
```

Comment:
1. It's not a TODO really, it's just a note
2. Could you please put it onto the same line `val cbor = "2.9.10"  //  See snowplow/snowplow/issues/4266

Reply:
Sure, will do now